src/routes/__root.tsx

```diff
@@ -0,0 +1,39 @@
+import { createRootRoute, Link, Outlet } from "@tanstack/react-router";
+import { TanStackRouterDevtools } from "@tanstack/router-devtools";
+
+// It's the layout component
+export const Route = createRootRoute({
+  component: () => (
+    <>
+      <div className="p-2 flex gap-2">
+        <Link to="/" className="[&.active]:font-bold">
+          Home
+        </Link>{" "}
+        <Link to="/dashboard" className="[&.active]:font-bold">
+          Dashboard
+        </Link>
+        <Link to="/attendance-management" className="[&.active]:font-bold">
+          Attendance Management
+        </Link>
+        <Link to="/document-management" className="[&.active]:font-bold">
+          Document Management
+        </Link>
+        <Link to="/employee-management" className="[&.active]:font-bold">
+          Employee Management
+        </Link>
+        <Link to="/payroll-management" className="[&.active]:font-bold">
+          Payroll Management
+        </Link>
+        <Link to="/recruitment" className="[&.active]:font-bold">
+          Recruitment
+        </Link>
+        <Link to="/time-tracking" className="[&.active]:font-bold">
+          Time Tracking
+        </Link>
+      </div>
+      <hr />
+      <Outlet />
+      <TanStackRouterDevtools />
```

Comment:
If we can only render this on dev, I feel like that would be amazing.

Reply:
I don't know what that means, would you be so kind as to spearhead the explanation of your request

Reply:
Node (and browser) usually keep track of different environments (e.g. dev, staging, prod) under the environment variable called `NODE_ENV`. We can access this using `process.env.NODE_ENV`. Vite handles this using [modes](https://vite.dev/guide/env-and-mode#modes).

In this case, the conditional render would look something like:
`import.meta.env.MODE === "development" && <TanStackRouterDevtools />`

(you need to configure your npm dev script to run with mode as development)

Reply:
Done!